insights/parsers/tests/test_getconf_pagesize.py

```diff
@@ -0,0 +1,20 @@
+from insights.parsers.getconf_pagesize import GetconfPageSize
+from insights.tests import context_wrap
+
+GETCONFPAGESIZE1 = """
+4096
+""".strip()
+
+GETCONFPAGESIZE2 = """
+16384
+""".strip()
+
+
+class Testgetconfpagesize():
```

Comment:
Would you mind to remove the `class` and change to test it with functions directly?